spring-integration-hazelcast/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+log4j.rootCategory=WARN, stdout
```

Comment:
This file has to be changed to `log4j2-test.xml`.
We don't support Log4J 1 any more.